examples/netcore/Program.cs

```diff
@@ -13,11 +13,11 @@ class Program {
         // Your Node URL (replace your-node with the name of your node)
         private const string nodeUrl = "https://data-api.your-node.dataspace-node.com";
         // Your Hub URL (replace your-hub with the name of your hub)
-        private const string tokenUrl = "https://id.your-hub.dataspace-hub.com/connect/token";
+        private const string tokenUrl = "https://auth.your-hub.dataspace-hub.com/auth/realms/your-node/protocol/openid-connect/token";
         // the scope setting defines what parts of an api / endpoints should be accessible
         // in this case it is either data-api-endpoint for the Data API or data-api-endpoint trust-api-endpoint for the Trust API.
         // The Trust API requires both scopes since it comes with the DSK Agent Integrated which passes through values to the Data API
-        private const string scope = "data-api-endpoint";
+        private const string scope = "profile email data-api node-id";
         // The following two settings can be found in the DataSpace Admin App (Profile -> Administration)
         private const string clientId = "<your-node-specific-api-client>";

```

Comment:
Not the node specific client anymore (which was previously the id of the node). It's now either data-api or trust-api as clientId (and they also have their secrets).